source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -29,10 +30,10 @@ func main() {
 		}
 	}()
 
-	// Ping the primary
+	// Send a ping to confirm successful connection
```

Comment:
Suggestion:

I think this needs an article since "successful connection" is an object rather than the name of a status.

```suggestion
	// Send a ping to confirm a successful connection
```